corehq/apps/zapier/api/v0_5.py

```diff
@@ -162,3 +162,54 @@ def obj_get_list(self, bundle, **kwargs):
 
     class Meta(BaseZapierCustomFieldResource.Meta):
         resource_name = 'custom_fields_case'
+
+
+class ZapierCustomActionFieldCaseResource(BaseZapierCustomFieldResource):
+    """
+        Generates custom action fields for updating or creating a case.  When creating a case, users must provide
+        a new case name, as well as optionally providing values for the case properties.  When updating a case,
+        users must provide a case id, as well as optionally providing new values for the case properties.
+    """
+    def obj_get_list(self, bundle, **kwargs):
+        custom_fields = []
+        domain = bundle.request.GET.get('domain')
+        case_type = bundle.request.GET.get('case_type')
```

Comment:
would it ever happen that case_type or domain is blank? Might be worth validating that all your inputs are correct here and sending an error back to Zapier if not.